src/main/kotlin/racingcar/vo/Position.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.vo
+
+private const val MINIMUM_POSITION_VALUE = 0
+private const val MOVE_UNIT = 1
+
+class Position(var value: Int = MINIMUM_POSITION_VALUE) {
```

Comment:
```suggestion
class Position(private var _value: Int = MINIMUM_POSITION_VALUE) {
    
    val value : Int
        get() = _value
```

외부에서 value에 대한 값을 수정하면 어떻게 될까요? 

저는 value가 외부에서는 수정이 불가능해야 한다고 생각합니다. 이를 처리하기 위해서는 어떻게 해야 할까요?

제가 보여준 backing_property를 씀으로써 _가 들어간 value는 setter 용도로, 외부에서 보여주기 위한 getter는 _가 없는 value로 사용 할 수가 있습니다.

제가 좋은 자료 드릴 테니 꼭 읽어보시길 바랍니다! 

[backing-properties](https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties)

Reply:
오 val을 내부에 설정하고, getter의 값을 var값을 줌으로써 내부에서 var값은 가변적이지만,
setter가 외부에 막히게 되고, getter만 열어주는 상태가 되는군요.
코틀린에서 자주 사용되는 컨벤션일까요?

```
class Position(value: Int = MINIMUM_POSITION_VALUE) {
    var value = value
        private set

```

보다 위의 방법이 더 자주 사용되는 컨벤션일까요?

Reply:
이 부분에 대해서는 스타일 차이가 아닐까 해요.

저는 위의 방법을 더 추천하고 있습니다 😊